ir-engine/etherealengine-archive
# IR-2153: replace scope checks with project permission checks

## Summary

IR-2153 & IR-2154

Replace scope checks in studio for project permission checks. 

## Subtasks Checklist

## Breaking Changes

## References
closes #_insert number here_

## QA Steps
- Log in with a user that doesn't have the location:write scope. 
- In the dashboard click on any scene in any project
- Once in the studio click on the Publish button on the upper right corner 
- Indicate a name for the location
- Click on submit. it should create a location.
